src/pages/splashSettings/SplashSettings.jsx

```diff
@@ -226,39 +222,22 @@ export default function SplashSettings() {
 
           <Button
             onClick={() => {
-              /* Prepare custom fields objects to send to backend */
-              Object.values(customFields).forEach(customFieldKey => {
-                const fields = currentValues[customFieldKey];
-                if (!fields) {
-                  currentValues[customFieldKey] = {
-                    definitions: [],
-                  };
-                } else {
-                  const newFields = get(
-                    fields,
-                    'definitions',
-                    [],
-                  ).map(field => {
-                    const choices = get(field, ['schema', 'choices']);
-                    if (!choices) return field;
-                    return {
-                      ...field,
-                      options: choices.map(choice => choice.label),
-                    };
-                  });
-
-                  currentValues[customFieldKey] = {
-                    definitions: newFields,
-                  };
-                }
-              });
               putSiteSettings({ data: currentValues });
```

Comment:
Rather than sending potentially 4 messages here, it would be far more efficient to add the  splashVideoPostData etc to currentValues and send them as one message

Reply:
super excited about this if it works!

Reply:
Of course it works, I wrote the code :laughing: